esmvalcore/preprocessor/_area.py

```diff
@@ -156,19 +161,32 @@ def tile_grid_areas(cube, fx_files):
                 continue
             logger.info('Attempting to load %s from file: %s', key, fx_file)
             fx_cube = iris.load_cube(fx_file)
-
             grid_areas = fx_cube.core_data()
```

Comment:
Perhaps it would be better to use `.lazy_data()` to guarantee a dask array compatible with the dask functions?